### DISCUSSION TITLE
我有一个想法，我无法参与断贷，但是我可以出钱帮助那些需要因为烂尾选择断贷导致生活有困难的人。

### DISCUSSION THREAD

**Discussion by guoyongchang** [WeNeedHome/SummaryOfLoanSuspension#872] [Ideas]
### 嘻嘻。
------

我有买河南的房子，目前没有停工烂尾，但是我总感觉离我不远了。

即便停工了我也会依然坚持还贷，因为不还贷给我带来的影响会更大。

我不愿这个铁拳砸到我身上的时候没人为我说话，我愿意拿出 1000CNY/月 来帮助那些因为烂尾生活有困难的人。

钱虽不多，但是我希望的是站在背后的，目前没有受到烂尾影响的人也能够站出来拿出一些东西来帮助顶在前面的同志！

现在有以下问题需要解决

1、谁来统筹这件事情

2、我的钱如何转到统筹者账户

-------

想到一个解决方案，走Web3的合约及多签确认。

具体实现是创建一个合约
这个合约没有所有者，没有管理者，然后往合约里面充钱的人拥有投票权（权重按照金额来）。

需要帮助的人去发申请，当投票人及权重>50%或者某个更高的权重同意时的时候合约会自动将代币打到需求者账户。
为了效率也可以按照申请金额降低同意百分比。 
也可以加一些额外的条件防止个别大户偷撸。

但是这个要所有人都会Web3

不知是否有Web3大佬来评定该方案是否可行

毕竟走国内，对接人风险太大，随便一个罪就给你兜进去了。

**Comment by guoyongchang** [comment]
你们快说话 你们不说话我好害怕啊

**Reply by xianyulao** [reply]
大佬买的河南哪里的房子啊，我买的郑州的，最近也准备停贷了，一是房子也停工了，二是老婆怀孕了需要用钱的地方多，没必要为了别人的过错承担责任，拿还贷的钱给老婆买点营养品，给未来几个月出生的宝宝买点奶粉他不香吗

**Reply by JWleo** [reply]
> 大佬买的河南哪里的房子啊，我买的郑州的，最近也准备停贷了，一是房子也停工了，二是老婆怀孕了需要用钱的地方多，没必要为了别人的过错承担责任，拿还贷的钱给老婆买点营养品，给未来几个月出生的宝宝买点奶粉他不香吗

断之前先转移好你的资产，不然强制给你交贷

**Reply by xianyulao** [reply]
> > 大佬买的河南哪里的房子啊，我买的郑州的，最近也准备停贷了，一是房子也停工了，二是老婆怀孕了需要用钱的地方多，没必要为了别人的过错承担责任，拿还贷的钱给老婆买点营养品，给未来几个月出生的宝宝买点奶粉他不香吗
> 
> 断之前先转移好你的资产，不然强制给你交贷

是滴是滴，先做好准备，等我们自己建的业主群里商量好了就集体断供，征信算个鸡巴，就算没问题，我这一个月还完贷款也没有能力再去干其他事了😂😂

**Reply by guoyongchang** [reply]
> 大佬买的河南哪里的房子啊，我买的郑州的，最近也准备停贷了，一是房子也停工了，二是老婆怀孕了需要用钱的地方多，没必要为了别人的过错承担责任，拿还贷的钱给老婆买点营养品，给未来几个月出生的宝宝买点奶粉他不香吗

俺洛阳的 大兄弟加油啊 JWleo大佬说的没错 得把资产转移好
单位里可以沟通下能走现金走现金吧 不然到时候钱进银行卡就被强制转走了

**Comment by Redlegen** [comment]
很有意义，顶一下，无产阶级help无产阶级

**Comment by imhappyok** [comment]
这个不难，一个小区搞一个微信企业群最大限额1w人，添加所有停贷户的户主作为成员，这样大概群成员最多会有3000余人，再加上所有愿意赞助该小区生活困难的义士，总人数应该不会超过1w的，再设置对公联名账户，刚搜了好像最多5个人，这样还是有可能5个人偷偷转移资金，其他人不知道，要是可以做到同时关联多个手机号或者邮箱，每次进账出账都有短信或是邮件提示就好了，有没有人知道这一块的？，如果把这个问题解决了就好了，规定这笔资金除了每月给困难者发补贴外，不允许其他使用，什么聚会啊，买矿泉水都不行，只有进账和月固定出账，简单明了，每个月入账和支出固定，然后还要避免有人冒领，这就需要其他验证方式了，肯定有人懂的，感觉这事要做好也不容易啊。。。

**Reply by Redlegen** [reply]
我个人觉得建整个小区的群太麻烦了，而且把整个小区的居民都拉进来不现实，可以把每个小区的负责人拉一个群，然后每个人负责负责统筹统计该小区需要帮助的人士并且提供一定的书面或者电子证明。可以成立一个类似工会或者协会的组织，专门负责管理捐款以及各类援助行为和支出等。这是在你的灵感基础上想到的办法。

**Reply by imhappyok** [reply]
可是可以，但是户主不能直接联系到捐款人可能会导致小区负责人多占，有些人拿不到或者是拿的少，必须给其他户主能联系到部分捐款人，当然只拉小区负责人也行，不过必须要让小区户主能直接联系到捐款人的途径，不然很可能出现漏洞导致腐败，因为这历来是个大问题，难以杜绝，只有透明才能避免

**Reply by Redlegen** [reply]
都要搞捐款了，肯定财务是第一大事，要做财务清单和政务清单等等，在程序上保证公平公正公开，这些应该是可以办到的。提这个方案主要是觉得可行性相对高一点，而且技术含量也不是很高，也方便群众监督什么的。

**Reply by z-huabao** [reply]
有点地下组织的味道了

**Reply by CKwuliuqi** [reply]
这样搞，我看很刑啊，不出俩月就进去了

**Comment by xincao9** [comment]
感觉杯水车薪，如果你来钱不是很容易的话。还是攒着为以后生活做保障

**Reply by imhappyok** [reply]
咋说呢，他应该生活有结余，所以可能出点力，有时候帮助一些困难的人也是很快乐的事，可惜我比较穷，要是富裕的话，也愿意伸出援手吧

**Comment by GithubRobot01** [comment]
很多人断贷并不是还不起，大部分是跟风，而且现在还没有到停贷的时候，到时候肯定有人偷偷还

**Reply by imhappyok** [reply]
除了多套房的，一套房还贷款大多生活还是比较艰难的，如果生活条件好的话，那就不赞助呗，也需要结合一些情况来看

**Reply by zhw173** [reply]
> 除了多套房的，一套房还贷款大多生活还是比较艰难的，如果生活条件好的话，那就不赞助呗，也需要结合一些情况来看

是啊，这几年农村不让建房，很多农民子女只能在县城买房，东拼西凑节衣缩食凑了首付，按揭怎么办，进工厂007，一年回家一次

**Comment by guoyongchang** [comment]
想到一个解决方案，走Web3的合约及多签确认。

具体实现方案就是创建一个合约
这个合约没有所有者，然后往合约里面充钱的人拥有投票权（权重按照金额来）。

需要帮助的人去发申请，当投票人及权重>50%或者某个更高的权重同意时的时候合约会自动将代币打到需求者账户。
为了效率也可以按照阈值降低同意百分比。 也可以加一些额外的条件防止个别大户偷撸。

但是这个要所有人都会web3

**Reply by guoyongchang** [reply]
这里有个点就是可以很轻易的验证申请人是否需要帮助，毕竟物理位置都是在一个小区的。

**Comment by JWleo** [comment]


**Reply by guoyongchang** [reply]
今日我不为人发声呐喊，明日又有谁会为我发声呐喊？

**Reply by imhappyok** [reply]
虽然我一方面觉得这些大量贷款的人傻，另一方面也觉得他们可怜，都是可怜人啊

**Reply by JWleo** [reply]
> 今日我不为人发声呐喊，明日又有谁会为我发声呐喊？

可以呐喊出声，只是暂时不用捐出你的血汗钱。这是社会问题，只能靠集体资助，个人的钱拿出来基本意义不大。知道你也是好心

**Reply by JWleo** [reply]
> 虽然我一方面觉得这些大量贷款的人傻，另一方面也觉得他们可怜，都是可怜人啊

同是社会打工人，相逢何必曾相识。辛辛相惜吧

**Reply by guoyongchang** [reply]
> 同是社会打工人，相逢何必曾相识。辛辛相惜吧

目前能有钱且敢出钱资助的团体，恐怕也只有洋人了。

但是估计洋人也懒得为这个烂摊子发声。

目前我个人认为可行方案还是得无产阶级自救

**Reply by zhaomingtao-fighting** [reply]
> 

主要是房价不涨，不然他们觉得你才傻呢

**Comment by imhappyok** [comment]
要是可以保证财务安全，是可行的，先看看有多少人愿意捐款吧，人数太少或是资金太少就难以启动了


| |
黄超
|
|
***@***.***
|

签名由 网易邮箱大师 定制

在2022年07月19日 10:46，JWleo 写道：

虽然我一方面觉得这些大量贷款的人傻，另一方面也觉得他们可怜，都是可怜人啊

同是社会打工人，相逢何必曾相识。辛辛相惜吧

**Comment by Accelerator6666** [comment]
根据抵押及售房合同合同部分信息，近期还贷流水表来审核吧。敏感信息自行处理掉上传审核。
这一环应该考虑加进去吧。
可以试试陪审团机制？每发起一个ticket，随机指定若干名审核人员？
但是信息的真实性验证还是有点难度。

**Comment by Redlegen** [comment]
我觉得合约没有所有者可以，没有管理者不太现实，捐助不是单纯筹钱然后打款就可以了，这其中有很多需要工作的环节，比如收集和整理捐助人名单及信息，审查是否满足捐助条件，甚至还有一些更加复杂的内容。所以如果真想做的话，也是一件不简单的事情。

**Reply by Redlegen** [reply]
而且需要帮助的人如果都困难到那种程度，大概率是不会主动在网上甚至墙外刷到这些救助信息的，所以肯定需要有人去主动发现需要救助的人或者宣传救助活动。这中间需要很多小伙伴一起来做工作的。

**Reply by CKwuliuqi** [reply]
行动第一步刚开始，就被网警发现然后线下被抓获，背负一个恶意串联颠覆国家ZQ的罪名

**Comment by xiejun946** [comment]
理论上停贷就是止损了，没有必要集资，最大的风险的征信带来的问题，而不是金钱生活上的

**Reply by Accelerator6666** [reply]
理论上是这样，停止还贷，宣布个人破产。但是国家希望能做到贷贷相传。

**Reply by CKwuliuqi** [reply]
怎么可能破产呢，如果你破产了银行的债务怎么办，这就会引发系统性金融风险，银行就直接GG了啊，企业可以破产是因为企业的金融风险已经转移了，被转移到居民身上了，居民破产了风险往哪转移？

**Comment by masami1912** [comment]
我停贷不是因为我没钱还，是因为我不想还了。

**Comment by chenhaizano** [comment]
这人呆湾骗子，鉴定完毕

**Reply by chenhaizano** [reply]
同时也是脑残智障人士，yue~~~~

**Reply by guoyongchang** [reply]
> 这人呆湾骗子，鉴定完毕

建议你点名指姓的@指定的人，你这样没有限定范围我会认为你在说我

**Comment by ppinkiris** [comment]
我觉得现在最迫切的需要是所有烂尾楼的业主集体停贷。
不太清楚目前比例大概是多少，有知道的家人可以说下。
比如按城市来，举例郑州。
保证烂尾楼盘的断贷率+供应商断贷率在90%以上，那离胜利就不远了。
但是怎么把这部分人组织起来呢？

还有就是得保证该市新期房楼盘一定没人买。
没买房的应跑尽跑，能润的润，不能润的换城市。
我周围同学的同事有河南的，听说已经发誓打死不回了。
总之重要的还是 大家得团结，再就是有组织才行。

最后最重要的一点：远离恨国党及台巴子转移视线，台巴子港灿恨国党回复我死一车亲妈。

**Reply by Accelerator6666** [reply]
问题是谁敢组织？颠覆国家罪怕不怕？国安法怕不怕？

**Reply by CKwuliuqi** [reply]
组织这个事，我看刑，当天组织当天进去

**Comment by Kamijoucen** [comment]
同样愿意，希望有人组织，并且公开钱款去向

**Comment by xiaer93** [comment]
关键没人敢组织，颠覆国家政权、寻衅滋事、1450、美分 等大帽子扣下来，就算有流水记录证据，也没有人相信承认。

**Comment by lenny0x** [comment]
业务理论上是可行，但是就技术来讲，需要有代币支撑，代币需要什么比例来进行现金置换，现金置换的手续费怎么来抽取，交易也需要看链的主网络是否有gas费，其中的环节挺多的。这是我对目前web3的一些理解，如有出入，希望大家能指出

**Comment by Jasonlsammy** [comment]
解决不了根源问题，都是无用功，甚至留下污点。有好心做好事，按道理是要支持，但是综合利弊还是建议不要做。
web3，我这边有团队，可代发币，可是资金的监管风险非常大，第一有案例被封过60w，直接还在冻结中，第二有案例卷跑路。

**Comment by clleo888** [comment]
楼主有爱心, 建议你随便到一个烂尾楼, 上楼梯找5户人家, 每家都加微信, 每个月每户转200就帮到忙了, 他们一定是比较艰难的, 不艰难会住进烂尾楼吗? 不要相信任何统筹, 在中国要点对点直接帮助才可行。

**Reply by JWleo** [reply]
红十字会：你在指我咯？

**Reply by clleo888** [reply]
> 红十字会：你在指我咯？

不只红十字会, 国内任何NGO都受控制打压, 不让你做真心帮人的事, 因为这样会影响党的权力。反而你去骗人, 党有方法让你生存, 你说这国家....!!!

**Reply by CKwuliuqi** [reply]
开玩笑，什么红十字会，什么XX工程，全是面子光鲜里子腐烂不堪，但凡能了解点里面内情的，三观都给你惊碎

**Reply by ksana404** [reply]
这个方法比较稳妥

**Comment by tingbiaolab** [comment]
跟钱沾边的都考验人性。

**Comment by Dreaming-Pepper** [comment]
@clleo888 这个老铁说的【点对点】没毛病，很切实的执行方法，我建议多加一条，范围缩小到你身边，比如同个小区、周边小区等等，不要跨太远，主要是审核【对点】会变麻烦。
若周边都没有，就去看网上那些曝光住烂尾楼，爬高楼挑水的人把。
再把这事情曝光出去，说有人给那些被曝光住烂尾楼的人钱，然后那些没拿到钱的人，大家就争先恐后去曝光自己。
断贷是自损800敌挣800的方法，只有二十四小时不间断曝光、卖惨、构陷那些“坏人”才是正解，要摆烂，大摆烂。

**Comment by Cegazhos** [comment]
从和谐角度看，我肯定是支持你的。但是 我不喜欢现在的中修，我觉得应该踩油门。

**Comment by RadioAc** [comment]
大家都开始讨论上技术上的实现问题了,没人考虑过这样做的敏感性吗

**Comment by guoyongchang** [comment]
新消息，我也要开始维权啦。

嘻嘻。

**Reply by JWleo** [reply]
大兄弟，扎心了

**Reply by ksana404** [reply]
出什么事了?

**Reply by jaychen1996-byte** [reply]
扎铁了，老心
